scrapers/scrapers/spiders/sehgal_motors_spider.py

```diff
@@ -0,0 +1,56 @@
+import scrapy
+
+
+class SehgalMotorSpider(scrapy.Spider):
+    
+    name = 'sehgalmotors'
+    sehgal_motors_url = 'https://www.sehgalmotors.pk/'
+
+    
+    def start_requests(self):
+        
+        yield scrapy.Request(url=self.sehgal_motors_url, callback=self.parse)
```

Comment:
No need to perform a start request in this case. We use start requests for the case where we have to do something custom before landing on the crawling page.